inc/saitypes.h

```diff
@@ -302,11 +302,7 @@ typedef enum _sai_object_type_t
     /** Must remain in last position */
     SAI_OBJECT_TYPE_MAX,
 
-    /** Custom range base value */
-    SAI_OBJECT_TYPE_CUSTOM_RANGE_START = 256,
```

Comment:
hi Kamil, just found that these attributes are removed. do we need to keep them but maybe with a different value?

Reply:
currenty those are not needed, and they were added to fix gcc error, do a git blame and find commit for that change, if we want to add those back, then they need to start as custom range for other enums 0x10000000